pkg/pillar/containerd/containerd.go

```diff
@@ -90,6 +92,11 @@ func InitContainerdClient() error {
 	}
 	if contentStore == nil {
 		contentStore = CtrdClient.ContentStore()
+		// see if we need to fine-tune default snapshotter based on what flavor of storage persist partition is
+		persistType, err := ioutil.ReadFile(eveStorageTypeFile)
+		if err == nil && string(persistType) == "zfs" {
```

Comment:
Might want to make this a `switch`/`case` statement; I figure we will have other types in the future? Maybe not?

Reply:
Also, what do we do if the file does not exist? We should fall back to the original `defaultSnapshotter`

Reply:
> Also, what do we do if the file does not exist? We should fall back to the original defaultSnapshotter

the original one is always "overlayfs" -- that didn't change -- so think about this snippet of code as an opportunistic override for now